From 79c1e72a150677fa7696a50c30428b77065be54d Mon Sep 17 00:00:00 2001
From: Danila Tikhonov <danila@jiaxyga.com>
Date: Fri, 8 Sep 2023 21:23:18 +0300
Subject: [PATCH 010/194] FROMLIST: dt-bindings: dmaengine: qcom: gpi: Add the
 SM7150 compatible

Add a compatible for the GPI DMA controller on SM7150. It uses the same
0x0 offset as SDM845.

Signed-off-by: Danila Tikhonov <danila@jiaxyga.com>
---
 Documentation/devicetree/bindings/dma/qcom,gpi.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/dma/qcom,gpi.yaml b/Documentation/devicetree/bindings/dma/qcom,gpi.yaml
index 54dca6232..76983f520 100644
--- a/Documentation/devicetree/bindings/dma/qcom,gpi.yaml
+++ b/Documentation/devicetree/bindings/dma/qcom,gpi.yaml
@@ -52,6 +52,7 @@ properties:
               - qcom,qcs615-gpi-dma
               - qcom,sdm670-gpi-dma
               - qcom,sm6125-gpi-dma
+              - qcom,sm7150-gpi-dma
               - qcom,sm8150-gpi-dma
               - qcom,sm8250-gpi-dma
           - const: qcom,sdm845-gpi-dma
-- 
2.43.0

